### PR TITLE
libarchive 3.6.2: rebuild against libxml2 2.13

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Destination channel:** main

### Links

- [PKG-5279](https://anaconda.atlassian.net/browse/PKG-5279) 

[PKG-5279]: https://anaconda.atlassian.net/browse/PKG-5279?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by JamesRobertsonGames — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -49,7 +49,7 @@ requirements:
     #  against liblzo."
     # - lzo
     - openssl {{ openssl }} # [not osx]
-    - libxml2 2.10
+    - libxml2 {{ libxml2 }}
     - zlib {{ zlib }}
     - zstd {{ zstd }}
   run:
```
Comment (danpetry) [inline recipe/meta.yaml:55]: `pin_compatible(`openssl`, x.y.z)` ? unless there's run_exports in openssl?

Comment (skupr-anaconda) [reply recipe/meta.yaml:55]: I think a plain openssl is enough. I didn't find the usage pin_compatible for openssl. Other recipes have openssl in `run` if `openssl {{ openssl }}` in `host`, for example, https://github.com/AnacondaRecipes/arrow-cpp-feedstock/blob/master/recipe/meta.yaml#L83

Comment (skupr-anaconda) [reply recipe/meta.yaml:55]: I replaced it with `openssl`

Comment (danpetry) [reply recipe/meta.yaml:55]: yep, there's a run_exports in the openssl package that will take care of compatibility

**Review by danpetry — APPROVED** [review]

**Pull request merged**